mws/mws.py

```diff
@@ -311,9 +311,9 @@ def make_request(self, extra_data, method="GET", **kwargs):
         headers.update(kwargs.get("extra_headers", {}))
         body = kwargs.get("body", "")
 
-        if method == "POST":
+        if method == "POST" and use_body_params:
             url = url_base
-            body += url_data
+            body = str(body) + url_data
```

Comment:
have you tested this work, don't we need to decode the binary data?

Reply:
in theory it will never be binary data unless we set the flag from other places too when calling this method. but we shouldn't because if it's binary the two payloads don't mix together. the authentication/merchant/etc. params are url encoded strings, so to combine with the supplied body arg, they have to be in the same format.